big5/README.md

```diff
@@ -68,6 +68,15 @@ This workload allows the following parameters to be specified using `--workload-
 * `target_throughput` (default: 2): Target throughput for each query operation in requests per second, use 0 or "" for no throughput throttling.
 * `warmup_iterations` (default: 100): Number of warmup query iterations prior to actual measurements commencing.
 
+NOTE: If disabling `target_throughput` when using OSB in a bash script, use `0` or do not specify quotations after `target_throughput`. See the following code for invalid and valid examples:
+```
+# In a bash script, this will be invalid
+--workload-params=target_throughput:""
+
+# In a bash script, these two lines will be valid
+--workload-params=target_throughput:0
+--workload-params=target_throughput:
+```
```

Comment:
This isn't entirely correct.  The effect with quotes and without quotes should be identical, since the shell consumes them.  There may be a problem if the command line has additional nested quotes, but that should be called out with additional clarity.